queries/javascript.fieldAccess.scm

```diff
@@ -0,0 +1,85 @@
+;; foo.bar
+;; ^^^
+(
+  (member_expression
+    object: (_) @fieldAccess
+  )
+  (#not-type? @fieldAccess call_expression member_expression subscript_expression)
+)
+
+;; foo().bar
+;; ^^^^^
+(
+  (member_expression
+    object: (call_expression
+      function: (_) @dummy
```

Comment:
wasn't sure what to call nodes like this where we're just using them to allow us to write a predicate that rejects the pattern.  We should probably standardize on something that allows multiple per pattern, because in the future I'd like to be strict about allowable capture names to detect typos in capture names, which today just result in scopes silently not working